contributors/guide/review-guidelines.md

```diff
@@ -27,127 +18,10 @@ description: |
   - [Checking out a PR](#checking-out-a-pr)
 - [Additional Resources](#additional-resources)
 
-
-## Tips for getting your PR reviewed
-
-
-### Use Tokens to signal the state of your PR
-
-Token phrases like `WIP` or `do not merge` signal to reviewers that the PR is
-potentially in a state that is not complete. This can save significant time for
-reviewers on prioritizing which PR should be reviewed. They can either hold off
-on reviewing altogether or will review the PR with the knowledge that it is not
-complete.
-
-
-### Reduce the scope of your PR
-
-Extremely large PRs take significant amounts of time to both review and evaluate
-for potential upstream/downstream impacts. If it is possible to achieve the
-desired goal over multiple smaller, more concise PR ("Pre-factor"), it decreases
-risk and will have a greater chance of being accepted.
-
-
-### Split your PR into logical smaller commits
-
-Multiple smaller commits that contain the logical pieces of work simplify the
-review process for others to review your code. These commits should be able to
-stand on their own and not necessarily depend on other things.
-
-**Example: Markdown**
-
-A PR that applies both formatting changes and content changes. If taken together
-in a single commit, it will appear to your reviewer that the entire document has
-changed and will require more time to compare and separate the actual content
-changes from the style changes.
-
-If content and style changes were made in separate commits, they could be reviewed
-independently for their changes.
-
-**Example: Generated code**
-
-A PR that has both authored code and generated code can be difficult to parse.
-If the authored code and generated code are in separate commits, the generated
-code can largely be ignored by the reviewer, and they can focus their time on
-the authored code.
```

Comment:
These examples might be worth integrating, they're 2 common scenarios that happen with PRs.